src/libs/Notification/PushNotification/index.native.js

```diff
@@ -109,12 +106,6 @@ function register(accountID) {
     // Regardless of the user's opt-in status, we still want to receive silent push notifications.
     Log.info(`[PUSH_NOTIFICATIONS] Subscribing to notifications for account ID ${accountID}`);
     UrbanAirship.setNamedUser(accountID.toString());
-
-    // When the user logged out and then logged in with a different account
-    // while the app is still in background, we must resubscribe to the report
-    // push notification in order to render the report click behaviour correctly
```

Comment:
I attempted to simplify this comment [here](https://github.com/Expensify/App/pull/14702/files#diff-571033c4e40d8aa5e2b60377b898d82d804e539b987033e6979e89352a567b8aR45)